config/hscConfig.py

```diff
@@ -2,3 +2,7 @@
 config.processCcd.isr.doAttachTransmissionCurve = False
 # these commissioning data do not have the correct header info to apply the stray light correction
 config.processCcd.isr.doStrayLight = False
+# Run meas_modelfit to compute CModel fluxes
```

Comment:
Should you explicitly
```
import lsst.meas.modelfit
```
here as well?

Reply:
I will just in case; obs_subaru just happens to be configured to load it by default (in config/*.py).